### ISSUE TITLE
Extension doesn't seem to activate automatically

### ISSUE THREAD

**Issue by nikomatsakis** [nikomatsakis/intorust.vscode#6]
I'm not sure why but I seem to have to run "IntoRust: Enable codelens" from the command palette to get things to show up.

**Comment by doc-jones** [comment]
I have a couple of questions on this. 
1. Based upon a couple of other example extensions I've seen the activationEvents value was simply [ ]. 
       So activationEvents: [ ] have you seen/tried this?
2. In package.json main is specified as "./dist/extension.js". 
       a. In the docs it is stated that "main" in package.json would be set to "./out/extension.js" for extensions in dev and  "./dist/extension.js" for production.  Is this extension in production?  Does it matter?

**Comment by nikomatsakis** [comment]
I believe that `[]` would mean "never activate unless explicitly used". I think that `["*"]` means "always active immediately". The former is too weak and the latter seems stronger than necessary. The current setting (activate when Rust code is used) seems right, but for some reason the codelens items don't show up. It might be something dumb like the default configuration being `false`.

**Comment by kwdev** [comment]
I wonder if it is a timing issue, I haven't had to run "IntoRust: Enable codelens". After opening the "Extension Development Host" window (is that what you're doing?), it seems to depend on how/when the CodeLensProvider.provideCodeLenses is triggered, when diagnostics are available, etc. Putting focus on the window usually triggers it, or editing the file.

**Comment by doc-jones** [comment]
> I believe that `[]` would mean "never activate unless explicitly used". I think that `["*"]` means "always active immediately". The former is too weak and the latter seems stronger than necessary. The current setting (activate when Rust code is used) seems right, but for some reason the codelens items don't show up. It might be something dumb like the default configuration being `false`.

Yes, I see the difference between [] and ["*"].  I'm still catching up on the extensions documentation.

**Comment by nikomatsakis** [comment]
Maybe I'm just impatient
